trio/_socket.py

```diff
@@ -549,6 +550,7 @@ def shutdown(self, flag):
         # empty list.
         assert len(gai_res) >= 1
         # Address is the last item in the first entry
+        # normed = gai_res[0][-1]
```

Comment:
I put it in because it does the same as your assignment but pylint doesn't complain. Was meant as a reminder to change this one day so we have one complaint less.
(actually this one was the last standing) But this should be very likely if at all a different PR. Right?

Reply:
Oh, does pylint complain about the unpacking assignment? That's weird, it seems more readable to me...

And yes, better to separate it out in any case.

Reply:
Huh, yeah, looks like a pylint bug – it's supposed to catch illegal code like `*a, *b = c`, but it also fires on valid code like this. Filed at https://github.com/PyCQA/pylint/issues/2513

Reply:
Reported 16 hours ago, fixed an hour ago, how do you do this? 
Besides I would have never considered this a bug in pylint. So amazing.
Anyways, removed the comment again.